ledger/participant-integration-api/src/main/scala/platform/apiserver/services/transaction/ApiTransactionService.scala

```diff
@@ -71,45 +77,25 @@ private[apiserver] final class ApiTransactionService private (
       logging.endInclusive(request.endInclusive),
       logging.parties(request.filter.filtersByParty.keys),
     ) { implicit loggingContext =>
-      val subscriptionId = subscriptionIdCounter.incrementAndGet().toString
-      logger.info(s"Received request for transaction subscription $subscriptionId: $request")
+      logger.info("Received request for transactions.")
```

Comment:
Any reason why we want to avoid logging the request?

Reply:
It's massive and the data is mostly in the logging context anyway.

Reply:
the `template_id` filter is not logged or is it?

Reply:
I agree with the removal of subscription id, it came from before open-sourcing of daml.

I cannot agree with the removal of the request. It is a piece of info we look very often when trying to understand what is going in the production systems. In fact I was looking at this info this morning with Tudor. It is logged only when the subscription is made, so very rarely compared to let's say transaction submission

If we want to shorten the wording of the log for "Received request for" we should then change it across the board. ATM all client requests share this wording, which makes grepping easy.

Reply:
By changing the production logback settings we may enable logging the requests for this particular service for a specific ledger or disable such log lines for a specific ledger via the same mechanism.
Wouldn't we be able to grep for all requests by simply grepping for 'request' under the package `com.daml.apiserver.services`?

Reply:
Can you explain which information you were looking at this morning? I am interested in whether this information was also in the logging context. I think that contains everything I would be looking at, but I can't speak for you.

Reply:
@SamirTalwar-DA Yes, that one is missing, together with the `ledgerId` (which is arguably useless) and `verbose` flag. But thinking better about it: instead of making the loggingContext fatter from the beginning, isn't it better to log one-offs with the `request` inlined?

Reply:
on the other hand, if you add the `template_id` filters to the `loggingContext`, the log line would have similar size as before with the extra penalty that you would log these extra fields all over the place downstream.

Reply:
Yeah, that's a good point. We definitely don't want this data in every log line.

We have two competing concerns:

1. @oliverse-da, among others, have asked for shorter log lines that don't contain unnecessary information. They have moved to multi-line logging which is incompatible with the Daml SDK without a lot of rewriting.
2. You, among others, require this information.
3. Our one-off logging information is unstructured, and hard to manipulate.

So here's what I suggest:

1. We move the request data to the structured logging context, but _only_ for the `info` line, after which it is discarded.
2. We look into correlation IDs for requests which allow us to correlate log lines.
3. In the future, we find a way to automatically log the entire request in a structured format (i.e. JSON), without explicit logging code.

What do you think?

Reply:
Sure, makes sense, but that log wasn't generating multi-lines. So what was the issue with it?

Reply:
These logs are generating long lines. By moving the data to the structured context, I'm hoping @oliverse-da can then output the data over multiple lines without compromising on your debugging needs.

Reply:
How do you feel about these log lines?

```
16:30:08.949 [in-memory-ledger-akka.actor.default-dispatcher-75] INFO  c.d.p.a.s.t.ApiTransactionService - Received request for transactions. , context: {startExclusive: "00000000000000950000000000000000", endInclusive: "%end%", filters: {TXNoContractKey-alpha-359797a68be5-party-1: "all-templates", TXNoContractKey-alpha-359797a68be5-party-0: "all-templates"}}
16:30:08.950 [in-memory-ledger-akka.actor.default-dispatcher-73] INFO  c.d.p.a.s.t.ApiTransactionService - Received request for transactions. , context: {startExclusive: "000000000000008d0000000000000000", endInclusive: "%end%", filters: {TXAgreementTextExplicit-alpha-359797a68be5-party-0: ["e1f0a0f5fef4773fe78916593f7ff9c7a8ca65f9f538192e3a5335d778cae60f:Test:Dummy"]}}
```

Reply:
Done, and I moved the logging context so it _only_ wraps the `info` line for each subscription request.